lib/logstash/outputs/bigquery/schema.rb

```diff
@@ -0,0 +1,86 @@
+require 'java'
+require 'logstash-output-google_bigquery_jars.rb'
+require 'logstash/json'
+
+module LogStash
+  module Outputs
+    module BigQuery
+      class Schema
+        include_package 'com.google.cloud.bigquery'
+
+        # Converts a CSV schema or JSON schema into a BigQuery Java Schema.
+        def self.parse_csv_or_json(csv_schema, json_schema)
```

Comment:
Where is this code used?

Reply:
This is used in google_bigquery.rb:177, I'm calling it at register so we fail early if the user inputs an invalid schema.

Reply:
Thanks. The diff was collapsed so Ctrl + F failed me :)